### DISCUSSION TITLE
cross-seed searched the torrent but never saved/injected but I can see on tracker it was snatched

### DISCUSSION THREAD

**Discussion by mouzzampk2014-zz** [cross-seed/cross-seed#108] [Q&A]
Hi, I am not sure what happened here but I just cannot find the solution for this.

I added a torrent in qbittorrent
```
today at 18:49:46 info: [4/21] Searching for The.Movie.2021.1080p.WEB-DL.DD5.1.H.264-ROCCaT
today at 18:49:49 warn: Found The.Movie.2021.1080p.WEB-DL.DD5.1.H.264-ROCCaT on Tracker 1 - skipping incomplete torrent
```
```
today at 19:10:54 info: [4/21] Searching for The.Movie.2021.1080p.WEB-DL.DD5.1.H.264-ROCCaT
today at 19:10:55 info: Found The.Movie.2021.1080p.WEB-DL.DD5.1.H.264-ROCCaT on Tracker 1 - injected
```

Jacket logs
`today at 18:49:47 07-19 18:49:47 Info Manual search in Tracker 1, Tracker 2 for The Movie 2021 => Found 13 releases 
`
`today at 19:10:54 07-19 19:10:54 Info Manual search in Tracker 1, Tracker 2 for The Movie 2021 => Found 13 releases (from cache) `

`cross-seed` injected a file from Tracker 1 but never injected from Tracker 2 but also I cannot find it in torrent-cache folder

So I have no clue what happened.

**Comment by mmgoodnow** [comment]
cross-seed snatches more torrents than it eventually deems valid matches, partially because the last couple match checks require more information than is available through Jackett's api. If having a lot of undownloaded snatches is a problem on one of your trackers, you should probably not use cross-seed with it.

**Reply by mouzzampk2014-zz** [reply]
Hmmm but I am little confused now the torrents cross-seed searched but didn't inject was exactly same and I manually ship hash check and worked without any issues.

I only noticed when torrents are in completed while cross-seed searched and jackett cache the results. So seem like thats whats happening?

**Reply by mmgoodnow** [reply]
What do the logs say?
